src/lib/validations/sql.js

```diff
@@ -167,11 +167,30 @@ function checkTablePrefixes( tables ) {
 	}
 }
 
+const openFile = ( filename, flags = 'r', mode = 666 ) => new Promise( ( resolve, reject ) => {
```

Comment:
Minor: from a readability perspective, I find it's much nicer to avoid arrow functions for this and go with proper functions instead.

```
function openFile( filename, flags = 'r', mode = 666 ) {
    return new Promise( ( resolve, reject ) => {
    ...
}
```

No change needed (since it's following existing conventions in this file), just wanted to point it out.

Reply:
No worries, I went ahead and changed it.  I don't love having to write that sort of wrapper anyway.  It's annoying that you can't wrap `fs.createReadStream`!

Thanks for the review, @mjangda!